unbound: enable tests

lovesegfault opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

baloo committed:
openssl3: init at 3.0.0

Signed-off-by: Arthur Gautier <baloo@superbaloo.net>

baloo committed:
openssl3: disable build-time feature detection

This enables KTLS support on linux.

Signed-off-by: Arthur Gautier <baloo@superbaloo.net>

baloo committed:
openssl: openssl3 is published under Apache License v2.0

Signed-off-by: Arthur Gautier <baloo@superbaloo.net>

lucasew committed:
cisco-packet-tracer: init at 7.3.1 and 8.0.1

Signed-off-by: lucasew <lucas59356@gmail.com>

NickCao committed:
v2ray-domain-list-community: init at 20211103073737

lovesegfault committed:
unbound: enable tests

andir reviewed (commented):
Since we have a NixOS test we should include that in the meta attributes?

vcunat reviewed (approved):
Good idea.  (and I agree with adding link to `nixosTests.unbound` as suggested)

GKasparov committed:
picom-next: init at unstable-2021-10-31

cpcloud committed:
google-cloud-cpp: add unit tests

alyssais committed:
pkgsMusl.connmanMinimal: fix build with Alpine patch

simonkampe committed:
snapmaker-luban: init at 4.0.3

GKasparov committed:
picom-next: add GKasparov to maintainer-list

r-burns committed:
perlPackages.XMLLibXML: fix build on darwin

figsoda committed:
cargo-deny: don't vendor openssl, fix license

lovesegfault committed:
nixos/tests/owncast: rewrite test

proux01 committed:
coqPackages.coqeal: 1.0.6 -> 1.1.0

erikarvstedt committed:
btcpayserver: 1.3.2 -> 1.3.3

cpcloud committed:
google-cloud-cpp: add comment about doInstallCheck and dependencies

dasJ committed:
Merge pull request #137004 from baloo/baloo/openssl/3.0.0-init

openssl3: init at 3.0.0

mattmelling committed:
python3Packages.tern: add packageurl dependency

figsoda committed:
selene: 0.14.0 -> 0.15.0

ambroisie committed:
prowlarr: 0.1.1.978 -> 0.1.1.1030

thiagokokada committed:
Merge pull request #144696 from GKasparov/picom-next

picom-next: init at unstable-2021-10-31

SuperSandro2000 committed:
Merge pull request #144741 from ambroisie/update-prowlarr

SuperSandro2000 committed:
Merge pull request #144739 from mattmelling/tern

SuperSandro2000 committed:
Merge pull request #144740 from figsoda/update-selene

SuperSandro2000 committed:
Merge pull request #144731 from erikarvstedt/btcpayserver-1.3.3

SuperSandro2000 committed:
Merge pull request #144156 from simonkampe/submit/snapmaker-luban

r3dl3g committed:
eclipses.plugins: Add Apache ivyde, iyvderv, ivyant and ivy plugins (#142407)

Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

legendofmiracles committed:
zerobin: fix build

SuperSandro2000 committed:
Merge pull request #140555 from NickCao/v2ray-domain-list-community

mkg20001 committed:
Merge pull request #144603 from cpcloud/google-cloud-cpp-add-tests

cpcloud committed:
arrow-cpp: add flight feature flag (#144609)

thiagokokada committed:
Merge pull request #144691 from legendofmiracles/zerobin

zerobin: fix build

thiagokokada committed:
Merge pull request #144686 from r-burns/xmllibxml

perlPackages.XMLLibXML: fix build on darwin

flokli committed:
xmedcon: fix meta.platforms

platforms.darwin and platforms.linux are both lists. They need to be
concatenated, otherwise they're not recognized as supported platforms.

avdv committed:
azuredatastudio: 1.17.1 -> 1.33.0 (#143078)

flokli committed:
xmedcon: 0.21.0 -> 0.21.2

in release 0.21.2 @ 20-Jul-2021 (Gtk+3)

    Small addition in our speedometer CSS theme: window titlebar is now in dark mode too
    Learned how to leave out annoying autotools reconfigure rules from Makefiles via AM_MAINTAINER_MODE macro
        Why? working with most recent autotools version <> older autotools version on most systems <> users are not necessarily developers ...
    Improvements for Wayland (CentOS Stream / OpenSUSE Tumbleweed)
        Menu "Online info" core dumps due to "toplevel" window problem
        Check "Gdk-CRITICAL gdk_monitor_get_workarea: assertion 'GDK_IS_MONITOR ()' failed"
        gtk_window_set_transient_for(window, parent)
        Not working GBC-widget (gamma, brightness, contrast)

in release 0.21.1 @ 01-May-2021 (Gtk+3)

    Require libpng >= 1.6.0 for Ubuntu's sake
    Added in GUI resizing to fifth and quint
    Secured two sprintf() statements
    Updated Inno Setup file for Gtk+3 dependencies
    Fresh update of application icon
    GIF89a: write "XMedCon" in application block

zimbatm committed:
Merge pull request #144751 from flokli/xmedcon-bump

xmedcon: 0.21.0 -> 0.21.2, fix meta.platforms

bobby285271 committed:
Merge pull request #135789 from lucasew/package/cisco-packet-tracer

cisco-packet-tracer: init at 7.3.1 and 8.0.1

lovesegfault committed:
Merge pull request #144703 from lovesegfault/fix-owncast-tests

doronbehar committed:
Merge pull request #144754 from alyssais/connman-musl

lovesegfault commented:
Picking this up, added
